packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -24,9 +24,10 @@
   "categories": ["Programming Languages"],
   "dependencies": {
     "@salesforce/salesforcedx-utils-vscode": "44.8.0",
+    "@salesforce/eslint-config-lwc": "^0.2.0",
```

Comment:
@muenzpraeger - Is this the version that we finalized on for estlint-config-lwc? Or do we need to publish a newer version?

Reply:
I see from `npm view ... @salesforce/eslint-config-lwc versions` that the latest is indeed `0.2.0` which was published three days ago so this seems right but I just want to confirm.